kubernetes_env/benchmark/benchmark.py

```diff
@@ -97,38 +134,75 @@ def plot(files):
                  ci=False,
                  marker='o')
     plt.title(title)
-    timestamp = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(time.time()));
-    plot_name = "-".join(names) + timestamp
+    timestamp = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime(time.time()))
+    plot_name = "-".join(filters) + timestamp
     util.check_dir(GRAPHS_DIR)
     plt.savefig(f"{GRAPHS_DIR}/{plot_name}.png")
     log.info("Finished plotting. Check out the graphs directory!")
     return util.EXIT_SUCCESS
 
-
-def run_fortio(platform, threads, qps, run_time, file_name):
+def run_fortio(url, platform, threads, qps, run_time, file_name):
     util.check_dir(DATA_DIR)
-    _, _, gateway_url = kube_env.get_gateway_info(platform)
     output_file = str(DATA_DIR.joinpath(f"{file_name}.json"))
     fortio_dir = str(FORTIO_DIR)
     cmd = f"{fortio_dir} "
     cmd += f"load -c {threads} -qps {qps} -jitter -t {run_time}s -json {output_file} "
-    cmd += f"http://{gateway_url}/productpage"
+    cmd += f"{url}"
     with open(output_file, "w") as f:
         fortio_res = util.exec_process(cmd,
                                        stdout=subprocess.PIPE,
                                        stderr=subprocess.PIPE)
         return fortio_res
 
 
+def burst_loop(url, threads, qps, run_time, queue):
```

Comment:
Hmm we should probably take a look at the traffic this one causes. For example with Grafana. That is pretty easy.

Reply:
This one: https://github.com/grafana/grafana?